MQ2AutoSize.cpp

```diff
@@ -1,627 +1,1387 @@
-// MQ2AutoSize.cpp : Resize spawns by distance or whole zone (client only)
-//
-// 06/09/2009: Fixed for changes to the eqgame function
-//				 added corpse option and autosave -pms
-// 02/09/2009: added parameters, merc, npc & everything spawn options,
-//				 bug fixes and code cleanup - pms
-// 06/28/2008: finds its own offset - ieatacid
-//
-// version 0.9.3 (by Psycotic)
-// v1.0 - Eqmule 07-22-2016 - Added string safety.
-//
-//////////////////////////////////////////////////////////////////////////////
-// Usage:
-//	This plugin will automatically resize configured spawns to the specified
-//	size. You can configure it to only resize within a specific range and then
-//	resize back to normal when your distance moves out of that range.
-//	Current default range is set to 50 and may be changed via INI or cmd line
-//	NOTE:  These effects are CLIENT SIDE ONLY!
-//
-// Commands:
-//  /autosize				  - Toggles zone-wide  AutoSize on/off
-//  /autosize dist			- Toggles distance-based AutoSize on/off
-//  /autosize pc			  - Toggles AutoSize PC spawn types
-//  /autosize npc			 - Toggles AutoSize NPC spawn types
-//  /autosize pets			- Toggles AutoSize pet spawn types
-//  /autosize mercs		  - Toggles AutoSize mercenary spawn types
-//  /autosize mounts		 - Toggles AutoSize mounted player spawn types
-//  /autosize corpse		 - Toggles AutoSize corpse spawn types
-//  /autosize target		 - Resizes your target to sizetarget size
-//  /autosize everything	- Toggles AutoSize all spawn types
-//  /autosize self			- Toggles AutoSize for your character
-//  /autosize range #		- Sets range for distance-based AutoSize
-//
-//  (Valid sizes 1 to 250)
-//  /autosize size #		 - Sets default size for "everything"
-//  /autosize sizepc #	  - Sets size for PC spawn types
-//  /autosize sizenpc #	 - Sets size for NPC spawn types
-//  /autosize sizepets #	- Sets size for pet spawn types
-//  /autosize sizetarget # - Sets size for target parameter
-//  /autosize sizemercs #  - Sets size for mercenary spawn types
-//  /autosize sizemounts # - Sets size for mounted player spawn types
-//  /autosize sizecorpse # - Sets size for corpse spawn types
-//  /autosize sizeself #	- Sets size for your character
-//
-//  /autosize status		 - Display current plugin settings to chatwnd
-//  /autosize help			- Display command syntax to chatwnd
-//  /autosize save			- Save settings to INI file (auto on plugin unload)
-//  /autosize load			- Load settings from INI file (auto on plugin load)
-//  /autosize autosave	  - Automatically save settings to INI file when an option is toggled or size is set
-//
-//////////////////////////////////////////////////////////////////////////////
+// MQ2AutoSize.cpp : Resize spawns by distance (client only)
 
 #include <mq/Plugin.h>
 
+// Constants
 const char* MODULE_NAME = "MQ2AutoSize";
```

Comment:
`mqplugin::PluginName` exists

Reply:
So that I make one change, what are you suggesting here. Are you asking me to remove the reference (original), are you asking me to rather put "MQ2AutoSize" into the PreSetup?